### PR TITLE
Change Theme

### PR DESCRIPTION
**Pull request by safwanAzman**
feat: add change theme function store in localstorage

### CODE REVIEW

File: app/Livewire/Layout/Navbar.php [hunk 1]
```diff
@@ -8,6 +8,32 @@
...
+        ['bg-cyan-600', 'Cyan'],
+        ['bg-sky-600', 'Sky'],
+        ['bg-blue-600', 'Blue'],
+        ['bg-indigo-600', 'Indigo'],
+        ['bg-violet-600', 'Violet'],
+        ['bg-purple-600', 'Purple'],
+        ['bg-fuchsia-600', 'Fuchsia'],
+        ['bg-pink-600', 'Pink'],
+        ['bg-rose-600', 'Rose']
+
```
Comment (nazirulfitrifauzi) [inline app/Livewire/Layout/Navbar.php:34]: buang space2 xpakai ni @safwanAzman

File: public/js/colorThemes.js [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+        200: '#fecdd3',
+        300: '#fda4af',
+        400: '#fb7185',
+        500: '#f43f5e',
+        600: '#e11d48',
+        700: '#be123c',
+        800: '#9f1239',
+        900: '#881337',
+    },
+
```
Comment (nazirulfitrifauzi) [inline public/js/colorThemes.js:267]: sini pon same buang space xpakai @safwanAzman

File: resources/views/livewire/layout/navbar.blade.php [hunk 3]
```diff
@@ -90,11 +97,14 @@ class="flex items-center px-4 py-2 text-sm font-medium text-gray-500 rounded-lg
                             </div>
                         </div>
                     </div>
+
```
Comment (nazirulfitrifauzi) [inline resources/views/livewire/layout/navbar.blade.php:100]: buang dua2 space @safwanAzman

**Review by nazirulfitrifauzi — CHANGES_REQUESTED** [review]
minor changes need fixing

File: public/js/colorThemes.js (hunk 2 above)
Comment (safwanAzman) [reply public/js/colorThemes.js]: done ✅

**Pull request merged**